CMakeLists.txt

```diff
@@ -249,7 +249,11 @@ else()
     endif()
 
     # For MSVC builds, this puts all debug symbols file in the same directory.
-    set(CMAKE_PDB_OUTPUT_DIRECTORY "${REACTOS_BINARY_DIR}/msvc_pdb")
+    if (MSVC)
+        set(CMAKE_PDB_OUTPUT_DIRECTORY "${REACTOS_BINARY_DIR}/msvc_pdb")
+    elseif(SEPARATE_DBG)
```

Comment:
It must be defined on cmake invocation. It's used when you want to produce separate debug symbols. I think I might be the only one to use it.

Reply:
When you first invoke configure.cmd/sh it first tries to find the available compiler it's going to use, before creating the directories etc... so that's why I was thinking there may exist a cmake variable that gets initialized / from which one can derive a new cmake variable with a shortened version of the compiler name (msvc/gcc/clang/...), that we then can use here to create the name of the symbols directory.

Reply:
Ah OK. I don't think suh a variable exists, I might be wrong though.

Reply:
I think the separation should be done by symbols format. Something like `symbols_pdb` and `symbols_dwarf`.
Does `CMAKE_PDB_OUTPUT_DIRECTORY` called like that for all compilers?

Reply:
@HBelusca we define such variables ourselves https://github.com/reactos/reactos/blob/2928084e2f86f1b2b39214dfaf872e76c0561650/sdk/cmake/config.cmake#L39-L50

Reply:
Out of this PR